conda_vendor/conda_vendor.py

```diff
@@ -222,131 +503,156 @@ def get_conda_platform(platform=sys.platform, custom_platform=None) -> str:
-        click.echo(click.style(f"Channel: {pkg['channel']}\nPackage: {pkg['fn']}\nURL: {pkg['url']}\nSHA256: {pkg['sha256']}\nSubdirectory: {pkg['subdir']}\nTimestamp: {pkg['timestamp']}", fg='yellow'))
-        click.echo(click.style("========================================================================", fg='blue', bold=True))
 
-    # patch repodata.json for each channel + subdir
-    channels = list(set(channels))
-    subdirs = list(set(subdirs))
-    for channel in channels:
-        for subdir in subdirs:
-            if subdir in channel:
-                click.echo(click.style(f"Reconstructing repodata.json with Hotfix for {subdir} using {channel}/repodata.json", bold=True, fg='red'))
-                reconstruct_repodata_json(f"{channel}/repodata.json", vendored_dir_path / subdir, fetch_action_packages)
+###########################################################################
+#                                                                         #
+#                    conda-vendor vendor                                  #
+#                                                                         #
+###########################################################################
 
-@click.group()
-@click.version_option(__version__)
-def main() -> None:
-    """Display help and usage for subcommands, use: conda-vendor [COMMAND] --help"""
-    pass
 
-@click.command("vendor", help="Vendor dependencies into a local channel, given an environment file")
+@click.command(
+    "vendor",
+    help="Vendor dependencies into a local channel, given an environment file",
+)
 @click.option(
-    "--file",
-    default=None,
-    help="Path to environment.yaml")
+    "-f", "--file", default=None, help="Path to solvable environment.yaml"
+)
 @click.option(
+    "-s",
     "--solver",
     default="conda",
-    help="Solver to use. conda, mamba, micromamba")
+    help="Solver to use. Examples: conda, mamba, micromamba",
+)
 @click.option(
-    "--platform",
     "-p",
-    default=get_conda_platform(),
-    help="Platform to solve for.")
+    "--platform",
+    default=_get_conda_platform(),
+    help="Platform to solve for.",
+)
 @click.option(
     "--dry-run",
     default=False,
-    help="Dry Run. Doesn't Download Packages - Returns Formatted JSON of FETCH Action Packages")
+    is_flag=True,
+    help="Dry Run. Doesn't Download Packages - Returns Formatted JSON of FETCH Action Packages",
+)
 @click.option(
     "--ironbank-gen",
     default=False,
-    help="Save IronBank Resources 'ib_manifest.yaml' in current directory")
-def vendor(file,solver, platform, dry_run, ironbank_gen):
+    is_flag=True,
+    help="Save IronBank Resources 'ib_manifest.yaml' in current directory",
+)
+def vendor(
+    file: str, solver: str, platform: str, dry_run: bool, ironbank_gen: bool
+):
+    """Main etry point to vendor a file.  This will (in the general case)
+    use conda-lock to solve the environment specified in an environment_yaml
```

Comment:
etry spelled wrong

Reply:
etry is spelled right.  entry is spelled wrong :). fixing

Reply:
fixed